### PR TITLE
JavaScript Problems

### PR DESCRIPTION
**Pull request by ZdravkoK88**

### CODE REVIEW

File: JavaScript/Solutions/03.Simple-conditions-exam-problems/Firm.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+function firm([projectHours, availableDays, workers]) {
+
```
Comment (KristianMariyanov) [inline JavaScript/Solutions/03.Simple-conditions-exam-problems/Firm.js:2]: remove line

File: JavaScript/Solutions/03.Simple-conditions-exam-problems/Firm.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    let workDays = availableDays * 0.90;
+    let overtimeHours = (workDays * workers) * 2;
+    let workHours = (workDays * workers) * 8;
+    let totalHours = Math.floor(workHours + overtimeHours);
+
+    if (projectHours <= totalHours) {
+        console.log(`Yes!${totalHours - projectHours} hours left.`);
+    }
+    else {
```
Comment (KristianMariyanov) [inline JavaScript/Solutions/03.Simple-conditions-exam-problems/Firm.js:11]: `else {` must be on the line above.
Make this changes in all files

File: JavaScript/Solutions/03.Simple-conditions-exam-problems/TransportPrice.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    }
+    else if (distance < 100) {
+        price = distance * 0.09;
+    }
+    else {
+        price = distance * 0.06;
+    }
+    console.log(price);
+}
+
```
Comment (KristianMariyanov) [inline JavaScript/Solutions/03.Simple-conditions-exam-problems/TransportPrice.js:24]: remove empty lines

File: JavaScript/Solutions/03.Simple-conditions-exam-problems/PoolPipes.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+function poolPipes([volume, pipe1, pipe2, hours]) {
+
```
Comment (KristianMariyanov) [inline JavaScript/Solutions/03.Simple-conditions-exam-problems/PoolPipes.js:2]: remove all empty lines at the beginning of the functions

**Review by KristianMariyanov — CHANGES_REQUESTED** [review]

**Comment by Teodor92** [comment]
General note: most of the variables can be changed to const-s as they do not change after initialization. Not sure if the const-s are covered by the contents of the book.

https://softwareengineering.stackexchange.com/questions/278652/how-much-should-i-be-using-let-vs-const-in-es6

**Pull request merged**